synapse/config/homeserver.py

```diff
@@ -35,6 +35,7 @@
 from .workers import WorkerConfig
 from .push import PushConfig
 from .spam_checker import SpamCheckerConfig
+from .groups import GroupsConfig
```

Comment:
no groups.py :(

Reply:
That would explain why jenkins is unhappy......